### DISCUSSION TITLE
Bazel Build failing for latest develop on my local machine

### DISCUSSION THREAD

**Discussion by yash10019coder** [oppia/oppia-android#4357] [Q&A (Other)] [answered]
When I try to build the app using bazel using command
`bazel build //:oppia`

I get this error
![image](https://user-images.githubusercontent.com/76404817/168444318-a0bbf4c2-786b-46c9-a55b-bf263187b0a8.png)
`ubuntu@ubuntu-linux:~/opensource/oppia-android$ bazel build //:oppia
INFO: Analyzed target //:oppia (0 packages loaded, 0 targets configured).
INFO: Found 1 target...
ERROR: /home/ubuntu/.cache/bazel/_bazel_ubuntu/0d63f9c40fd590ec5c67d8ca9eb79dca/external/maven/BUILD:1923:11: AarResourcesExtractor external/maven/_aar/unzipped/resources/androidx_lifecycle_lifecycle_runtime failed: (Exit 127): aar_resources_extractor failed: error executing command bazel-out/host/bin/external/bazel_tools/tools/android/aar_resources_extractor --input_aar ... (remaining 9 argument(s) skipped)

Use --sandbox_debug to see verbose messages from the sandbox
/usr/bin/env: 'python': No such file or directory
Target //:oppia failed to build
Use --verbose_failures to see the command lines of failed build steps.
INFO: Elapsed time: 0.407s, Critical Path: 0.17s
INFO: 16 processes: 16 internal.
FAILED: Build did NOT complete successfully`

My develop branch is up to date and merged with latest develop.

**Comment by yash10019coder** [comment, marked as answer]
python3 was installed and bazel was searching for python
so I installed python-is-python3 package
`sudo apt install python-is-python3`

which fixes my problem.

**Reply by BenHenning** [reply]
FWIW, this should be fixed once we can upgrade to Bazel 5.0 since it properly supports Python 3.
